Nodejs/Product/Profiling/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All these changes and files were generated by package manager when I added the new deps.